tests/atest/rules/misc/inconsistent_assignment/autodetect/expected_output.txt

```diff
@@ -1 +1 @@
-autodetect${/}test.robot:30:23 [W] 0909 The assignment sign is not consistent within the file. Expected '' but got '=' instead
+autodetect${/}test.robot:30:15 [W] 0909 The assignment sign is not consistent within the file. Expected '' but got '=' instead
```

Comment:
Where the new message shows?

The code:
```
    ${var}    ${var2}=    Keyword
```
Previous one showed at ``=`` and new one shows at the beginning of the second var?

Reply:
Yes. I found that such small one-character-long underlines are easy to miss and hard to hover.
In this case, the equal sign is connected to the second var, so I decided that it would be better to underline the whole `${var2}=`. Looks nicer from my perspective and I think should be more friendly for the user. I'd be happy to see another perspective.

Reply:
Ok I just noticed it underlines more than one character - this changes things. Yes it is fine. Should we update our expected files to include both start/end col and line? Because now we check only for start line and col. Or at least easier way to configure it when needed in our test framework - although we may have something in place already

Reply:
Yeah, see for example https://github.com/MarketSquare/robotframework-robocop/blob/2720f2ed06bc6bb20a9b7e5948d74f6b34ef89f9/tests/atest/rules/errors/invalid_argument/test_rule.py . We can use "end_col" to set different format (see TestRule class for format details)

Reply:
I think it's possible to mark the end of the issue (see `tests/atest/rules/errors/invalid_argument/expected_output.txt`), but I don't know if it's needed in the output.
Start of the issue is enough for the user to locate it. End seems a bit confusing, either to me.

Does it give any additional value to the user? We also need to make sure, that it does not break the underlining in IDEs.

Reply:
I dont mean to provide it to user, but include it on our tests - if the rule reports more than just single char (ie underline word) it could use extended format in the test so we can check if it is reporting end col/line correctly

Reply:
Too much effort for this PR, let's now just update the locations and then we can slowly update the tests if needed in separate PRs